test/unit/text-field/index.test.tsx

```diff
@@ -314,18 +314,20 @@ test('#adapter.label.hasLabel returns true if label exists', () => {
   assert.isTrue(wrapper.state().foundation.adapter_.hasLabel());
 });
 
-test('#adapter.label.hasLabel returns true if label exists', () => {
+test('#adapter.label.getLabelWidth returns offsetWidth of labelElement', () => {
   const div = document.createElement('div');
-  document.body.append(div);
-  const options = {attachTo: div};
+  document.body.appendChild(div);
+
   const wrapper = mount<TextField<HTMLInputElement>>(
     <TextField label='my label'>
       <Input />
     </TextField>,
-    options
+    {attachTo: div}
   );
-  assert.equal(wrapper.state().foundation.adapter_.getLabelWidth(), 56);
-  div.remove();
+  const labelElement = wrapper.instance().floatingLabelElement.current!.labelElement_;
+  assert.equal(wrapper.state().foundation.adapter_.getLabelWidth(), labelElement.current!.offsetWidth);
```

Comment:
was this change also for the warnings?

Reply:
When I tested on linux.
I saw that label width isn't 56. (https://github.com/material-components/material-components-web-react/pull/771#issuecomment-477524974)
And when initialLabelWidth isn't setting, getLabelWidth returns labelElement's offsetWidth.
So I think this code is better.